### DISCUSSION TITLE
Patchcore model has good accuracy but segmentation is terrible?

### DISCUSSION THREAD

**Discussion by lekshmijk** [open-edge-platform/anomalib#776] [Q&A]
Hello, I've been training a pcb category from the VISA dataset using patchcore and the results I get seem pretty good:

![Screenshot from 2022-12-08 17-48-19](https://user-images.githubusercontent.com/45484074/206445920-2d440dac-2a1f-4623-82fe-5c435af824b6.png)

However, when I look at the generated mask and heat map, it's..not that good?
![004](https://user-images.githubusercontent.com/45484074/206446116-15bf9650-bd48-4a7a-9f20-c29aa55bd286.png)
![003](https://user-images.githubusercontent.com/45484074/206446367-60801c02-2cf5-4605-9408-249ecac9a64d.png)

Can someone please suggest a way to get better segmentation results? Thank you so much in advance!

**Comment by kikyostudypython** [comment]
hello，Have you solved the problem？I had the same problem
